integreat_compass/core/settings.py

```diff
@@ -70,15 +69,6 @@
 
 WSGI_APPLICATION = "integreat_compass.wsgi.application"
 
-
-# Database
-# https://docs.djangoproject.com/en/4.1/ref/settings/#databases
-
-DATABASES = {
-    "default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}
-}
-
-
```

Comment:
A default database setting should be provided in any case, no matter whether it's overwritten in docker_settings or not.